.travis.yml

```diff
@@ -1,8 +1,8 @@
 language: node_js
 node_js:
-  - 6
-  - 4
   - 0.12
+  - 4
+  - 6
```

Comment:
Re-ordered these. It seems it's easiest to break 0.12, so I put it up front for the quickest response.